src/graph/GoExecutor.cpp

```diff
@@ -1408,5 +1468,73 @@ SupportedType GoExecutor::getPropTypeFromInterim(const std::string &prop) const
     return index_->getColumnType(prop);
 }
 
+nebula::cpp2::SupportedType GoExecutor::calculateExprType(Expression* exp) const {
```

Comment:
If the schema cache of graphd and the storaged are different, maybe has problem.

Reply:
Yes,  we use the schema in graphd. 
And the problem also existed in current master branch, because different storaged maybe have different versions schema when updating it.